### PR TITLE
chore: migrate to cli-table3

### PR DESCRIPTION
**Pull request by DanielRuf**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
   },
   "dependencies": {
     "chalk": "^2.1.0",
-    "cli-table2": "^0.2.0",
+    "cli-table3": "^0.5.0",
```
Comment (rwjblue) [inline package.json]: I think you will need to use 0.4.0 or update `.travis.yml`/`engines`/etc to drop Node 4.

Comment (DanielRuf) [reply package.json]: I think dropping Node.js 4 is better as it reached its EOL.

Comment (rwjblue) [reply package.json]: Totally agree, but it generally requires a major version bump so I figured I'd point out the option...

Comment (Turbo87) [reply package.json]: I tend to agree. Let's us v0.4.0 for now, and then we can drop Node 4 in a separate step :)

Comment (DanielRuf) [reply package.json]: Changed it to 0.4.0.

**Comment by stefanpenner** [comment]
we are about to do a major version bump, so I will rebase this now :)

**Pull request merged**